lib/MusicBrainz/Server/Data/Collection.pm

```diff
@@ -89,6 +89,15 @@ sub contains_entity {
         $collection_id, $id) ? 1 : 0;
 }
 
+sub non_empty {
```

Comment:
Given that we already have a generic utility method `non_empty` that tests for not-undef-and-not-the-empty-string, I’d prefer to have a different name for this